README.md

```diff
@@ -19,6 +19,34 @@ Once the website finishes launching the website will be available on port 80: [h
 Part of the deployment builds the docker containers used by the website, so it may take a few minutes to finish deployment.
 To stop the app run `docker-compose down`.
 
+### Certbot Setup on AWS
+Certificates can be installed to the EC2 VM using Certbot with Nginx on pip following [EFF instructions](https://certbot.eff.org/instructions?ws=nginx&os=pip).
+```bash
+sudo python3 -m venv /opt/certbot/
+sudo /opt/certbot/bin/pip install --upgrade pip
+sudo /opt/certbot/bin/pip install certbot certbot-nginx
+sudo ln -s /opt/certbot/bin/certbot /usr/bin/certbot
+```
+And to install Nginx on the Amazon Linux 2023 (following [instructions](https://awswithatiq.com/how-to-install-nginx-in-amazon-linux-2023/)) and run Certbot:
+```bash
+docker-compose down
+sudo dnf update -y
+sudo dnf install nginx -y
+sudo systemctl start nginx
```

Comment:
Is it necessary to temporarily install nginx and run it on the host VM?
What happens if you don't start nginx?

Reply:
The alternative to installing nginx on the host is to configure Certbot to work with the Dockerized nginx. Either way would work fine, but if neither is done, Certbot won't be able to validate our control of the domain.

Reply:
OK, sounds good!